src/mgr/PyModule.cc

```diff
@@ -371,6 +371,7 @@ int PyModule::load(PyThreadState *pMainThreadState)
       return r;
     }
 
+    register_options(pClass);
```

Comment:
Don't we want this to return if `register_options` fails?

Reply:
I didn't check because we wouldn't do anything different, and the function already logs a message to the log, which is enough to track down a regression